server.py

```
@@ -0,0 +1,255 @@
+        if websocket.open:
+             await send_to_client(websocket, {"type": "error", "data": {"message": "An unexpected server error occurred."}})
+    finally:
+        # The `async with` statement handles session.close() automatically.
+        # No need for explicit live_session.close() unless it was created outside `async with`.
+        logger.info(f"Client {websocket.remote_address} disconnected. Live session closed if active.")
+
+# --- Main Execution ---
+async def main():
+    api_key = os.environ.get("GOOGLE_API_KEY")
+    if not api_key:
+        logger.critical("CRITICAL: GOOGLE_API_KEY environment variable not set. Server cannot start.")
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)